### ISSUE TITLE
[Auto Blog] Replace Google Docs: browser-based editors that need no account

### ISSUE THREAD

**Issue by linisme** [nologin-tools/nologin.tools#5]
@claude Please write a blog post for nologin.tools based on the instructions below.

## Topic
Replace Google Docs: browser-based editors that need no account

## Category
tool-alternatives (Tool Alternatives)

## Trending Context
- No relevant trending topics found today. Use the suggested topic above.

## Existing Posts (DO NOT duplicate)
- welcome: "Welcome to nologin.tools" [tags: announcement, privacy]

## Available Tools for Internal Links
- /tool/nologin-tools — nologin.tools (A curated directory of tools and services that work without )
- /tool/led-run — led.run (A browser-based display toolkit that turns any screen into a)
- /tool/tmp-tf — tmp.tf (A temporary clipboard tool that lets you sync text and conte)
- /tool/til-re — til.re (A URL-based time tool that lets you create countdowns, count)
- /tool/excalidraw-com — Excalidraw (Virtual whiteboard for sketching hand-drawn like diagrams. S)
- /tool/photopea-com — Photopea (Advanced online image editor that supports PSD, XCF, Sketch,)
- /tool/tldraw-com — tldraw (A collaborative digital whiteboard with a clean, minimal int)
- /tool/squoosh-app — Squoosh (Google-built image compression tool that runs entirely in th)
- /tool/svgedit-netlify-app-editor-index-html — SVG Edit (Fast, web-based, JavaScript-driven SVG drawing editor. Creat)
- /tool/remove-bg — remove.bg (AI-powered tool that automatically removes backgrounds from )
- /tool/hemingwayapp-com — Hemingway Editor (Writing assistant that highlights complex sentences, passive)
- /tool/languagetool-org — LanguageTool (Multilingual grammar, style, and spell checker that supports)
- /tool/dillinger-io — Dillinger (Cloud-enabled, mobile-ready, offline-storage compatible Mark)
- /tool/regex101-com — Regex101 (Online regex tester and debugger for PHP, PCRE, Python, Gola)
- /tool/caniuse-com — Can I Use (Up-to-date browser support tables for web technologies inclu)
- /tool/devdocs-io — DevDocs (Combines multiple API documentations in a fast, organized, a)
- /tool/carbon-now-sh — Carbon (Create and share beautiful images of your source code. Choos)
- /tool/jsonformatter-org — JSON Formatter (Online JSON formatter, validator, and viewer. Paste or uploa)
- /tool/tools-pdf24-org-en — PDF24 Tools (Free and easy-to-use online PDF tools: merge, split, compres)
- /tool/tinywow-com — TinyWow (Collection of free online tools for PDF, video, image, and f)
- /tool/pomofocus-io — Pomofocus (Customizable Pomodoro timer that works on desktop and mobile)
- /tool/omnicalculator-com — Omni Calculator (Collection of over 3,000 free calculators for math, physics,)
- /tool/ezgif-com — ezGIF (Online GIF maker and image editor. Create, resize, crop, rev)
- /tool/audiotrimmer-com — Audio Trimmer (Simple online tool to trim audio files. Upload your audio, s)
- /tool/convertio-co — Convertio (Advanced online file converter supporting 300+ formats. Conv)
- /tool/haveibeenpwned-com — Have I Been Pwned (Check if your email address or phone number has been comprom)
- /tool/temp-mail-org — Temp Mail (Disposable temporary email service. Get an instant anonymous)
- /tool/privacytests-org — PrivacyTests (Open-source tests of web browser privacy. Compares tracking )
- /tool/csvjson-com — CSVJSON (Online tool for converting between CSV, JSON, and SQL format)
- /tool/jsoncrack-com — JSON Crack (Visualize JSON data into interactive graphs. Seamlessly expl)
- /tool/tableconvert-com — TableConvert (Convert between table formats: Excel, CSV, JSON, HTML, SQL, )
- /tool/meet-jit-si — Jitsi Meet (Free, open-source video conferencing that works directly in )
- /tool/desmos-com-calculator — Desmos (Advanced graphing calculator implemented as an interactive w)
- /tool/wolframalpha-com — WolframAlpha (Computational knowledge engine that answers factual question)
- /tool/geogebra-org-calculator — GeoGebra (Free dynamic mathematics software for all levels of educatio)
- /tool/xe-com-currencyconverter — XE Currency Converter (World-trusted currency converter with live exchange rates. C)
- /tool/investor-gov-financial-tools-calculators-calculators-compound-interest-calculator — Compound Interest Calculator (Official SEC compound interest calculator. Estimate how your)
- /tool/wise-com-gb-currency-converter — Wise Currency Converter (Currency converter showing the real, mid-market exchange rat)
- /tool/quickstart-to — quickstart.to (A community-driven, open source collection of quick-start gu)
- /tool/coolors-co — Coolors (Generate harmonious color palettes instantly. Press spacebar)
- /tool/jakearchibald-github-io-svgomg — SVGOMG (Optimize and compress SVG files with a visual interface. Bui)
- /tool/favicon-io — Favicon.io (Generate favicons from text, images, or emojis. Download in )
- /tool/tinypng-com — TinyPNG (Smart lossy compression for PNG, JPEG, and WebP images. Redu)
- /tool/app-diagrams-net — Diagrams.net (Create flowcharts, UML diagrams, network diagrams, and more.)
- /tool/gchq-github-io-cyberchef — CyberChef (The Cyber Swiss Army Knife — encode, decode, encrypt, compre)
- /tool/typescriptlang-org-play — TypeScript Playground (Write, compile, and run TypeScript code in the browser. Offi)
- /tool/crontab-guru — Crontab Guru (Visual cron expression editor with human-readable explanatio)
- /tool/bundlephobia-com — Bundlephobia (Find the cost of adding an npm package to your bundle. View )
- /tool/explainshell-com — ExplainShell (Parse and explain shell commands. Paste any command line to )
- /tool/webhook-site — Webhook.site (Instantly test and debug webhooks and HTTP requests. Get a u)
- /tool/it-tools-tech — IT Tools (Collection of 70+ developer utilities: hash generators, enco)
- /tool/ray-so — Ray.so (Create beautiful code screenshots with customizable themes, )
- /tool/fast-com — Fast.com (Simple internet speed test by Netflix. Open the page and it )
- /tool/worldtimebuddy-com — WorldTimeBuddy (Visual time zone converter for scheduling across time zones.)
- /tool/excalideck-com — Excalideck (Create hand-drawn style presentation slides using the Excali)
- /tool/unscreen-com — Unscreen (Automatically remove video backgrounds with AI. Upload a vid)
- /tool/vocalremover-org — Vocalremover.org (AI-powered vocal remover that separates vocals from instrume)
- /tool/browserleaks-com — BrowserLeaks (Comprehensive browser fingerprinting test suite. Check what )
- /tool/coveryourtracks-eff-org — Cover Your Tracks (Test how well your browser protects you from tracking. By th)
- /tool/virustotal-com — VirusTotal (Scan files and URLs for malware using 70+ antivirus engines.)
- /tool/pairdrop-net — PairDrop (Share files instantly with nearby devices on your local netw)
- /tool/privnote-com — PrivNote (Send self-destructing encrypted notes. The message is automa)
- /tool/sharedrop-io — ShareDrop (P2P file sharing in your browser. Like Apple AirDrop but wor)
- /tool/db-fiddle-com — DB Fiddle (Run SQL queries online against MySQL, PostgreSQL, and SQLite)
- /tool/mockaroo-com — Mockaroo (Generate realistic test data in CSV, JSON, SQL, and Excel fo)
- /tool/flexboxfroggy-com — Flexbox Froggy (Learn CSS Flexbox through a fun frog-themed game. Complete 2)
- /tool/cssgridgarden-com — CSS Grid Garden (Learn CSS Grid layout by tending a virtual garden. Water you)
- /tool/numbeo-com — Numbeo (World's largest cost of living database. Compare prices, ren)
- /tool/themeasureofaplan-com — The Measure of a Plan (Beautiful personal finance calculators for retirement, inves)
- /tool/godbolt-org — Compiler Explorer (Interactive compiler explorer that shows assembly output for)
- /tool/play-rust-lang-org — Rust Playground (Official Rust online compiler and execution environment. Wri)
- /tool/go-dev-play — Go Playground (Official Go online compiler and execution environment. Write)
- /tool/hoppscotch-io — Hoppscotch (Open-source API development ecosystem. Test REST, GraphQL, W)
- /tool/astexplorer-net — AST Explorer (Visualize and explore Abstract Syntax Trees for many languag)
- /tool/mermaid-live — Mermaid Live Editor (Create flowcharts, sequence diagrams, Gantt charts, and more)
- /tool/caniemail-com — Can I Email (Check HTML and CSS support across email clients. Like Can I )
- /tool/npmtrends-com — npm trends (Compare npm package download trends over time. See package s)
- /tool/shields-io — Shields.io (Generate quality badges for your README and documentation. H)
- /tool/httpbin-org — httpbin (Simple HTTP request and response service for testing. Inspec)
- /tool/prettier-io-playground — Prettier Playground (Preview Prettier code formatting results online. Experiment )
- /tool/wave-webaim-org — WAVE (Web accessibility evaluation tool by WebAIM. Enter any URL t)
- /tool/metatags-io — Metatags.io (Preview and generate social media meta tags. See how your pa)
- /tool/haikei-app — Haikei (Generate unique SVG background shapes — waves, blobs, gradie)
- /tool/cssgradient-io — CSS Gradient (Visual CSS gradient generator with a beautiful UI. Create li)
- /tool/getwaves-io — Get Waves (Generate beautiful SVG wave shapes for section dividers. Cus)
- /tool/realtimecolors-com — Realtime Colors (Preview color palettes on a real page layout in real time. S)
- /tool/colorhunt-co — Color Hunt (Curated collection of beautiful color palettes. Browse trend)
- /tool/neumorphism-io — Neumorphism.io (Visual neumorphism CSS generator. Adjust size, radius, dista)
- /tool/fontjoy-com — FontJoy (AI-powered font pairing generator using deep learning. Gener)
- /tool/typescale-com — Type Scale (Visual type scale calculator for building harmonious typogra)
- /tool/uicolors-app — UIColors (Generate Tailwind CSS color scales from any color. Get a com)
- /tool/happyhues-co — Happy Hues (Color palette inspiration with real-world context. See how c)
- /tool/cubic-bezier-com — Cubic Bezier (Visual CSS cubic-bezier timing function editor. Drag control)
- /tool/stackedit-io — StackEdit (Full-featured in-browser Markdown editor. Sync with Google D)
- /tool/rentry-co — Rentry (Markdown-powered paste service. Write, publish, and share fo)
- /tool/monkeytype-com — MonkeyType (Minimalist, customizable typing speed test. Track WPM, accur)
- /tool/zenpen-io — ZenPen (Minimal distraction-free writing environment. Just open and )
- /tool/visualgo-net — VisuAlgo (Animated visualizations of data structures and algorithms. S)
- /tool/flukeout-github-io — CSS Diner (Learn CSS selectors by playing a dinner-themed game. Complet)
- /tool/pythontutor-com — Python Tutor (Visualize code execution step by step. See variables, stack )
- /tool/regexlearn-com — RegexLearn (Interactive regex tutorial with step-by-step lessons. Learn )
- /tool/mystery-knightlab-com — SQL Murder Mystery (Learn SQL by solving a murder mystery. Use SELECT, JOIN, WHE)
- /tool/sortvisualizer-com — Sort Visualizer (Watch sorting algorithms in action with animated bar charts.)
- /tool/dnsleaktest-com — DNS Leak Test (Check if your DNS requests are leaking outside your VPN tunn)
- /tool/ipleak-net — IPLeak (Comprehensive leak detection: IP address, DNS servers, WebRT)
- /tool/securityheaders-com — Security Headers (Analyze HTTP security headers of any website. Get a grade an)
- /tool/rawgraphs-io — RAWGraphs (Open-source data visualization framework. Paste your data, c)
- /tool/transform-tools — transform.tools (Multi-format code and data transformer. Convert between JSON)
- /tool/geojson-io — GeoJSON.io (Create, edit, and visualize GeoJSON data on an interactive m)
- /tool/markmap-js-org — Markmap (Convert Markdown to interactive mind maps. Write structured )
- /tool/photomosh-com — PhotoMosh (Apply glitch effects to images and videos. 27 mosh effects i)
- /tool/animista-net — Animista (Collection of ready-to-use CSS animations. Preview, customiz)
- /tool/mp3cut-net — MP3 Cutter (Cut and trim audio files online. Support for MP3, WAV, OGG, )
- /tool/svgrepo-com — SVG Repo (Browse and download 500,000+ free SVG icons and vectors. Sea)
- /tool/speed-cloudflare-com — Cloudflare Speed Test (Comprehensive internet speed and latency test by Cloudflare.)
- /tool/musicforprogramming-net — Music For Programming (Curated ambient and electronic music mixes designed to aid c)
- /tool/copychar-cc — CopyChar (One-click copy for special characters, symbols, and emoji. B)
- /tool/time-is — Time.is (Exact current time synced with atomic clocks. World clock, t)
- /tool/archive-today — Archive.today (Save permanent snapshots of web pages. Archive any URL to pr)
- /tool/qrcode-monkey-com — QR Code Monkey (Create custom QR codes with logos, colors, and shapes. High-)
- /tool/yopass-se — Yopass (Share secrets securely with end-to-end encryption. Messages )
- /tool/wormhole-app — Wormhole (End-to-end encrypted file transfer up to 10 GB. Files are en)
- /tool/ficalc-app — FICalc (Financial independence retirement calculator. Simulate withd)
- /tool/financier-io — Financier (Offline-first personal budget manager inspired by YNAB. Enve)
- /tool/duck-ai — DuckDuckGo AI Chat (Privacy-focused AI chat powered by Claude, Llama, Mistral, a)
- /tool/chatgpt-com — ChatGPT (OpenAI's AI assistant available without login. Handles writi)
- /tool/perplexity-ai — Perplexity (AI-powered search engine that provides accurate answers with)
- /tool/z-ai — Z.ai Chat (Free AI chatbot powered by GLM-5 and GLM-4.7 from Zhipu AI. )
- /tool/deepl-com-translator — DeepL Translator (AI-powered translation engine with superior accuracy across )
- /tool/quillbot-com — QuillBot (AI paraphrasing tool that rewrites text for clarity and styl)
- /tool/craiyon-com — Craiyon (Free AI image generator that creates 9 images from text prom)
- /tool/teachablemachine-withgoogle-com — Teachable Machine (Google's no-code tool for training machine learning models i)
- /tool/playground-tensorflow-org — TensorFlow Playground (Interactive neural network visualization in the browser. Exp)
- /tool/goblin-tools — Goblin.tools (AI-powered task management collection designed for neurodive)
- /tool/askmeanythi-ng — AskMeAnything (The simplest way to host an AMA session. Create a space, sha)
- /tool/cookies-extractor-chekun-me — Cookies Extractor (A simple yet powerful Chrome extension that helps you extrac)
- /tool/siteage-org — SiteAge (Trace the origins of any website through the Internet Archiv)
- /tool/next-tools-dev — next-tools (This is a tool website forked from it-tools, containing 100+)
- /tool/ps-keepmot-com — All-in-one tool for image cropping, collages, photo walls, and poster creation. (Tool Overview
A free online collage and image editing tool. )
- /tool/kuakua-app — Kuakua - Psychology & AI Tools for Well-Being (Kuakua.app is your AI-powered hub for mental health resource)

### File location
Save the blog post to `src/content/blog/{slug}.md` where `{slug}` is a short, URL-friendly version of the title (lowercase, hyphens, no special chars).

### Frontmatter format
```yaml
---
title: "Your Title Here"          # 50-70 characters
description: "A concise summary"  # 120-160 characters
publishedAt: 2026-03-10
author: "nologin.tools"
tags: ["tag1", "tag2"]            # 2-5 tags from: privacy, tools, tutorial, comparison, review, open-source, browser, guide, analysis, roundup, alternatives
featured: false
---
```

### Writing style (CRITICAL — zero AI feel)

**Banned phrases** — do NOT use any of these:
delve, landscape (as metaphor), "in this article we will", "it's worth noting", "in conclusion", "game-changer", "seamless", "leverage", "robust", "cutting-edge", "navigate" (as metaphor), "empower", "unlock", "comprehensive" (self-referential), "without further ado", "dive in", "in today's digital age", "harness", "revolutionize", "streamline", "elevate", "foster", "spearhead", "paradigm", "synergy"

**Do this instead:**
- Open with a hook: a surprising fact, a question, a short story, or a contradiction
- Mix sentence lengths — short punchy ones between longer ones
- Use active voice and address the reader as "you"; use "I" for opinions
- Name specific tools with URLs and concrete use cases
- Take a stance — "X is better than Y because..." is good
- Use parenthetical asides for personality (like this)
- Fragment sentences are fine. They add rhythm.
- 3-6 H2 headings, NEVER use "Introduction" or "Conclusion" as headings
- Paragraphs: 2-4 sentences max, minimize bullet lists
- Include at least one of: blockquote, code snippet, comparison table
- End with a forward-looking thought or call to action, not a summary

### Content requirements
- **Word count**: 1500-2500 words
- **Internal links**: Include at least 2 links to `/tool/{slug}` pages or `/blog/{slug}` posts from the lists above
- **External links**: Include at least 2 links to authoritative external sources
- **Keywords**: Naturally work in "no-login tools", "privacy-friendly", "without signup" throughout the post
- Research the topic using WebSearch to include accurate, current information

### Image
Use Unsplash API to download a relevant hero image:
```bash
mkdir -p public/blog/images/{slug}/
curl -s "https://api.unsplash.com/photos/random?query={relevant+search+term}&orientation=landscape&w=1200" \
  -H "Authorization: Client-ID $UNSPLASH_ACCESS_KEY" \
  | node -e "const d=JSON.parse(require('fs').readFileSync('/dev/stdin','utf8')); console.log(d.urls.regular)" \
  | xargs -I {} curl -sL -o public/blog/images/{slug}/hero.jpg "{}"
```
Then add the image right after frontmatter:
```markdown
![descriptive alt text](/blog/images/{slug}/hero.jpg)
```
If Unsplash API is unavailable (no API key or error), skip the image — the post is still valid without one.

### Publish steps
1. Create branch: `blog/2026-03-10-{slug}`
2. `git add src/content/blog/{slug}.md public/blog/images/` (if image exists)
3. `git commit -m "feat(blog): add post — {title}"`
4. `git push origin blog/2026-03-10-{slug}`
5. `gh pr create --title "Blog: {title}" --label "auto-blog" --base main`
6. `gh pr merge --auto --squash`
7. Close this issue with a comment summarizing what was written
